src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { useColonyContext } from '~hooks';
+import { useCopyToClipboard } from '~hooks/useCopyToClipboard';
+
+import { Heading3, Heading4 } from '~shared/Heading';
+import Button from '~v5/shared/Button';
+import Modal from '~v5/shared/Modal';
+
+const displayName = 'v5.common.Modals.InviteMembersModal';
+
+interface Props {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const MSG = defineMessages({
+  modalTitle: {
+    id: `${displayName}.modalTitle`,
+    defaultMessage: 'Invite members to the private beta',
+  },
+  modalDescription: {
+    id: `${displayName}.modalDescription`,
+    defaultMessage:
+      'You can invite up to 100 members to your Colony during the private beta using the invite link below.',
+  },
+  invitesUsed: {
+    id: `${displayName}.invitesUsed`,
+    defaultMessage: '{invitesUsed}/{invitesAvailable} invites used',
+  },
+  inviteLinkHeading: {
+    id: `${displayName}.inviteLinkHeading`,
+    defaultMessage: 'Your unique invite link:',
+  },
+  buttonText: {
+    id: `${displayName}.buttonText`,
+    defaultMessage: `{isCopied, select,
+      true {Link copied}
+      other {Copy link}
+    }`,
+  },
+});
+
+// @TODO: Add logic to get the real amount of invites available and used.
+const invitesAvailable = 100;
+const invitesUsed = 0;
+
+const InviteMembersModal = ({ isOpen, onClose }: Props) => {
+  const { colony } = useColonyContext();
+  const inviteLink = `app.colony.io/${colony?.name}/invite`;
```

Comment:
You can also get the correct invite code (which replaces the `invite` string in this path with https://github.com/JoinColony/colonyCDapp/pull/1207/commits/b4553e07139c0f8dbd12550f214c3e14b428beae